docs/running-notebooks/closing-active-sessions/index.md

```diff
@@ -30,3 +30,21 @@ click on the vertical ellipsis menu on the right-hand-side and select
 "Close Jupyter Notebook".
 
 ![ui-close-session](../../images/ui-close-session.png)
+
+### Auto-closure of sessions
+
+REANA provides an auto-closure feature for interactive sessions.
+Administrators can configure a maximum number of inactivity hours after
+which interactive sessions will be automatically closed.
+When you open an interactive session via the web interface,
+you will see a note inside the notification
+about the creation of the interactive session:
```

Comment:
```suggestion
When you open an interactive session via the web interface,
you will be informed about how long your inactive session will be kept open:
```
WDYT?

Reply:
Note: since we are adding the notification about the inactive session automatic closure also in `reana-client`, I removed "via the web interface".
I also changed "when you **open** an interactive session" to "when you **start** an interactive session", to avoid word repetition.

Reply:
> Note: since we are adding the notification about the inactive session automatic closure also in `reana-client`, I removed "via the web interface"

What do you think about adding also the reana-client example?